src/js/components/Heading/HeadingSkeleton.js

```diff
@@ -0,0 +1,33 @@
+import React, { forwardRef, useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Skeleton } from '../Skeleton';
+
+
+const HeadingSkeleton = forwardRef(
+  (
+    props,
```

Comment:
```suggestion
    {
    as,
    level,
    size
    },
```
can we destructure the props?

Reply:
definitely